README.md

```diff
@@ -379,17 +411,103 @@ Documentation Forthcoming
 
 ### Modules
 
-#### `loadModuleData`
+#### Loading Modules
 
-Documentation Forthcoming
+In both the Server and the Browser, the Holocron Modules loaded depends on which path is matched to a URL using the [child routes configuration](#routing), or when a React component uses the [Holocron API](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md).
```

Comment:
grammar nitpick: should be `the Holocron **Module** loaded depends` or `the Holocron Modules loaded **depend**`

Reply:
based on the fact that a matched path _can_ load multiple modules, I think it should be `the Holocron Modules loaded depend`

Reply:
Fixed in fb577c4